src/noun.substance

```diff
@@ -469,6 +469,22 @@ hs: limestone
 hyper: calcium_ion
 hypo: metal
 
+w: Calciarenite
```

Comment:
Por que inicial maiúsculas?

Reply:
Também fiquei em dúvida a respeito. GeolBA usa inicial maiúscula, segui o padrão deles. É um substantivo comum, mas é um nome de rocha?

I had doubts about it and just went along with GeolBA's reference. It is a common noun, but it is still a rock name?

Reply:
Ceylonite and pleonaste refer to the same synset. One is written with caps and the other isn't. What is the rule/guideline?

Reply:
Pois é, no nosso artigo http://arademaker.github.io/files/gwc-2018-chronostratigraphic.pdf lembro que identificamos (acho que algum texto da wikipedia dizia isso 

> The adjectives are capitalized when the subdivision is formally recognized, and lower case when not; thus “early Miocene” but “Early Jurassic

Para os nomes de rochas, uma boa pergunta é saber se existe mesmo alguma convenção.  Para `Calciarenite` não achei nenhuma ocorrencia em https://www.onepetro.org/ nem https://pubs.geoscienceworld.org. 

Mas para `Chalk` achei casos em ambos os sites e ocorrencias tanto em maiusculas quanto em minusculas. Se inspecionando os casos não emergir nenhum padrão, tendo a pensar em manter em minúsculas. Mas seria bom investigar se existe alguma convenção.

Reply:
Obviamente em alguns contextos o termo pode estar em maiúsculas por ser parte do nome de um local, formação etc. e então o nome da rocha não deveria ficar em maiúsculas.

Reply:
Realmente em http://wn.mybluemix.net/synset?id=15051503-n temos os dois termos o que reforça a idéia de que temos que entender porque o uso de maiúsculas.

Reply:
https://github.com/globalwordnet/english-wordnet/issues/247 para acompanhar

Reply:
Achei um [capítulo de livro](https://books.google.com.br/books?id=V7nUnYKmrxgC&lpg=PA263&ots=W2L4SJSgj_&dq=rock%20names%20lower%20case%20upper&hl=pt-BR&pg=PA263#v=onepage&q=rock%20names%20lower%20case%20upper&f=false) que diz que é para usar minúsculas. O cap[itulo trata apenas de rochas ígneas, mas tem a chancela da IUGS (International Union of Geological Sciences).

Estou procurando algum style guide deles. Por enquanto olhei um [artigo](http://www.episodes.org/journal/view.html?volume=42&number=4&spage=301&vmd=Full) qualquer do journal deles e as rochas estão em minúsculas.

Reply:
Segui a regra que alinhamos: tomar a wikipedia como referência de maiúscula/minúscula.